docs/shape_batch_size_and_layout.md

```diff
@@ -31,17 +31,33 @@ In case the model can't be reshaped, it will remain in the original parameters a
 will get an error. The model server will also report such problem in the logs.
 
 # Changing model input/output layout
-**DEPRECATED** TODO: Suggest converting model to NHWC model (possible with Model Optimizer in 2022.1).
+OpenVINO™ models can be exported to IR format with arbitrary input/output layout using Model Optimizer. Models which process image data are usually exported with NCHW or NHWC layout. Image transformation libraries like OpenCV or Pillow use NHWC layout. To minimize amount of overhead caused by transposition operation, it is suggested to export your model with NHWC layout.
```

Comment:
the layout can't be arbitrary. I would explain that starting from 2022.1, model optimizer by default preserve the layout of the original model. While the model had NWHC layout it will be preserved by default. Model optimizer can also export the model with additional transposition layout so to change to layout it needed.
Beside of that, the model sever can also change the model layout in runtime using --layout parameter.

Reply:
I would not recommend stating that by default model optimizer preserves layout of original model. This is plugin specific. Forwarded you an email from Ilya stating that they do not guarantee that.

From e-mail:
> Output<Node> contains information about tensor names, element type and shape and rt_info contains info about layout if it was specified by user via MO or runtime API. But cannot guarantee that all plugins restore layout information to the port after import_model

Reply:
The layout can be set to arbitrary, meaning not only NHWC or NHWC but any kind of layout combined with `N/C/H/W/D/?/...`.

`--layout` parameter is described below